docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
```

Comment:
That's unfortunate that slugs are not unique. I guess you can add user + slug to the endpoint like this though:
````
GET /api/v3/collections/collection/mozilla/webdeveloper/
````